packages/web-api-scan-runner/package.json

```diff
@@ -69,13 +67,14 @@
         "logger": "1.0.0",
         "moment": "^2.29.1",
         "p-limit": "^3.1.0",
-        "puppeteer": "^5.5.0",
+        "puppeteer": "^10.0.0",
         "reflect-metadata": "^0.1.13",
         "scanner-global-library": "1.0.0",
         "service-library": "1.0.0",
         "sha.js": "^2.4.11",
         "storage-documents": "1.0.0",
         "verror": "^1.10.1",
+        "yamlparser": "^0.0.2",
```

Comment:
where do we use `yamlparser`?

Reply:
Similar to playwright, this is an incorrectly declared dependency in a transitive dependency that is needed for building with webpack to work